appengine/queue/taskqueue.py

```diff
@@ -0,0 +1,397 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import logging
+import webapp2
```

Comment:
https://cloud.google.com/appengine/docs/standard/python/tools/built-in-libraries-27

I don't think it's needed